python/cli/prelude_cli/views/build.py

```diff
@@ -61,8 +61,8 @@ def create_template(template, name):
 
     if not test:
         Path(t['id']).mkdir(parents=True, exist_ok=True)
-        create_template(template='template.go', name=f'{t["id"]}.go')
         create_template(template='README.md', name='README.md')
+        create_template(template='template.go', name=f'{t["id"]}.go')
```

Comment:
lol

Reply:
haha there was a reason! if the `.go` file is created first, it tries to compile twice (once on each upload) - if it goes second, we'll only try to compile on the `.go` file upload

(also it is alphabetical which is a nice side effect)